zerver/management/commands/delete_realm.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+from typing import Any
+
+from django.conf import settings
+from django.core.management.base import CommandError
+
+from zerver.lib.management import ZulipBaseCommand
+from zerver.models import Message, UserProfile
+
+
+class Command(ZulipBaseCommand):
+    help = """Script to deactivate a realm."""
```

Comment:
This should say "permanently delete a realm.  Recommended only for removing realms used for testing; consider `deactivate_realm` instead", right?